src/components/emby-tabs/emby-tabs.js

```diff
@@ -18,13 +18,13 @@ define(['dom', 'scroller', 'browser', 'layoutManager', 'focusManager', 'register
 
     function onFocus(e) {
 
-        if (layoutManager.tv) {
+        /*if (layoutManager.tv) {
```

Comment:
I turned it off because when I focus on the profile settings in the top menu using the keyboard, this code activates the last tab. So, I leave activation by click only.

Reply:
It is fine to remove old code, sure, but you should do so by _deleting_ that code, not commenting it out.